tests/app_9/CMakeLists.txt

```diff
@@ -1,14 +1,20 @@
+cmake_minimum_required(VERSION 3.0)
+project (test D)
 include(UseDUnittest)
 
+set(CMAKE_BUILD_TYPE DEBUG)
+
+enable_testing()
+
 # TODO
 # how to rerun cmake if a new file is created
 # I don't want to miss new unittests
 
 # add all files
-file(GLOB files RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} "*.d")
+file(GLOB files RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} "module*.d")
```

Comment:
Globing is [not recommended](https://cmake.org/cmake/help/v3.10/command/file.html) anymore, simply specify the two files.
```
set(FILES
  ${CMAKE_CURRENT_SOURCE_DIR} "moduleA.d
  ${CMAKE_CURRENT_SOURCE_DIR} "moduleB.d)
```

Reply:
Just restored yours :-D